examples/MagicWeather/README.md

```diff
@@ -0,0 +1,79 @@
+# Magic Weather Android - RevenueCat Sample
+
+Magic Weather is a sample app demonstrating the proper methods for using RevenueCat's *Purchases* SDK. This sample uses only native platform components - no third-party SDKs other than the *Purchases* SDK.
+
+Sign up for a free RevenueCat account [here](https://www.revenuecat.com).
+
+## Requirements
+
+This sample uses:
+
+- Android Studio
+- Android (API 8.0+)
```

Comment:
I believe this is actually Android 8.0+, API level 26+: 
- [minSdkVersion defined here](https://github.com/RevenueCat/purchases-android/blob/main/examples/MagicWeather/app/build.gradle#L12)
- Android version to API level mapping defined here https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels
So I'd phrase this as either Android 8.0+ or API 26+ (or maybe both)

Reply:
Used both to match the Requirements section of the main readme